uint16 id 		# Object ID
string label            # Object label
float64 probability     # Confidence value in recognition
uint16 center_x       	# width center pixel of bb
uint16 center_y       	# height center height of bb
uint16 width_min       	# width min of bb (x left)
uint16 width_max      	# width max of bb (x right)
uint16 height_min      	# height min of bb (y top)
uint16 height_max       # height max of bb (y bottom)


